lib/components/ActionSheet/portal/index.tsx

```diff
@@ -13,6 +22,14 @@ interface PortalProps extends ComponentProps<"div"> {
     fullHeightOnOpen?: boolean;
 }
 
+type TCustomConstructor = React.JSXElementConstructor<{
+    handleClose: () => void;
+}>;
+type TChild = React.ReactElement<
+    { handleClose: () => void },
```

Comment:
nit-pick: just in order to not write the object twice :)
```
type TFooterChildProps = { handleClose: () => void };
type TFooterJSXConstructor = React.JSXElementConstructor<TFooterChildProps>;
type TFooterChild = React.ReactElement<TFooterChildProps, TFooterJSXConstructor>;
```

Reply:
Thank you